src/storage/Shards.sol

```diff
@@ -76,7 +76,6 @@ library ShardStore {
      * the shard info below 256 bit, so it can be stored in one single storage slot.
```

Comment:
The whole doc comment was just copy-paste from the `ShardInfo`'s one
@Lohann could you please update it properly?

Reply:
sure